mne/decoding/transformer.py

```diff
@@ -137,13 +137,13 @@ def inverse_transform(self, epochs_data, y=None):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
 
-        X = np.atleast_3d(epochs_data)
+        X = np.atleast_3d(epochs_data).copy()
 
         for key, this_pick in six.iteritems(self.picks_list_):
             if self.with_mean:
-                X[:, this_pick, :] += self.ch_mean_[key]
-            if self.with_std:
                 X[:, this_pick, :] *= self.std_[key]
+            if self.with_std:
+                X[:, this_pick, :] += self.ch_mean_[key]
```

Comment:
@jaeilepp - It seems this got merged without correction. Now self.with_mean does `X[:, this_pick, :] *= self.std_[key]` - probably there is no test for doing centering without scaling?